CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing
+
+This document provides guidelines for contributing to samples.
+
+## Dependencies
+
+The following dependencies must be installed on the development system:
+
+- [Docker Engine][docker-engine]
+- [Google Cloud SDK][google-cloud-sdk]
+- [make]
+
+## Integration Testing
+
+Integration tests are used to verify that the samples are actuatable 
+by Terraform. Tests are dynamically discovered based on directories in
+repo root and executed using the [blueprint-test](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test) framework.
+
+### Test Environment
+
+The easiest way to test the samples are in isolated test projects. The
+setup for projects are defined in [test/setup](./test/setup/)
+directory.
+
+To use this setup, you need to authenticate gcloud with an identity that has
+Project Creator access on a folder and the Billing Account User role on a billing account.
+
+Set the following environment variables.
+```
+export TF_VAR_org_id="your_org_id"
+export TF_VAR_folder_id="your_folder_id"
+export TF_VAR_billing_account="your_billing_account_id"
+```
+
+With these settings in place, you can prepare a test project using Docker:
+
```

Comment:
Add step to run:

gcloud auth application-default login

Reply:
When I run in Cloud Shell, this step isn't necessary because Cloud Shell runs this command automatically.

Reply:
I added this in just so that we have it documented if users are running locally.